litmus-portal/backend/graphql-server/pkg/database/mongodb/mongodb.go

```diff
@@ -0,0 +1,93 @@
+package database
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/graph/model"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var clusterCollection *mongo.Collection
+var backgroundContext = context.Background()
+var err error
+
+var collection = map[string]string{
+	"Cluster": "cluster-collection",
+}
+
+var dbName = "litmus"
+
+//DBInit initializes database connection
+func DBInit() error {
+	dbServer := os.Getenv("MONGODB_SERVER")
+	clientOptions := options.Client().ApplyURI("mongodb://" + dbServer)
+	client, err := mongo.Connect(backgroundContext, clientOptions)
+	if err != nil {
+		return err
+	}
+
+	ctx, _ := context.WithTimeout(backgroundContext, 20*time.Second)
+	// Check the connection
+	err = client.Ping(ctx, nil)
+	if err != nil {
+		return err
+	}  else {
+		log.Print("Connected To MONGODB")
+	}
+
+	clusterCollection = client.Database(dbName).Collection(collection["Cluster"])
+	return nil
+}
+
+func InsertCluster(cluster model.Cluster) error {
+	ctx, _ := context.WithTimeout(backgroundContext, 10*time.Second)
+	_, err := clusterCollection.InsertOne(ctx, cluster)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func GetCluster(cluster_id string) (model.Cluster, error) {
```

Comment:
Changing Find to FindOne operation

Reply:
You need to change the util functions too in the cluster package also.

Reply:
Already changed in this PR